drivers/led_strip/apa102.c

```diff
@@ -50,7 +60,8 @@ static int apa102_update(const struct device *dev, void *buf, size_t size)
 	return spi_write_dt(&config->bus, &tx);
 }
 
-static int apa102_update_rgb(const struct device *dev, struct led_rgb *pixels,
+static int apa102_update_rgb(const struct device *dev,
+			     struct led_rgb *pixels,
```

Comment:
Please, revert this cosmetic change. It is not related with the current patch. As a rule of thumb, don't pollute a patch with unrelated changes.